kaiax/gov/paramset.go

```diff
@@ -38,6 +41,7 @@ func GetDefaultGovernanceParamSet() *ParamSet {
 	for name, param := range Params {
 		err := ps.Set(name, param.DefaultValue)
 		if err != nil {
+			logger.Error("GetDefaultGovernanceParamSet error", "name", name, "value", param)
```

Comment:
Could you let me know if you intended to print the default value instead of printing the struct itself?
```
			logger.Error("GetDefaultGovernanceParamSet error", "name", name, "value", param.DefaultValue)
```